flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/JobGraph.java

```diff
@@ -530,6 +529,7 @@ public void uploadRequiredJarFiles(InetSocketAddress serverAddress) throws IOExc
 			}
 		}
 	}
+	*/
```

Comment:
I think its better to completely delete the method instead of commenting it out.